.github/workflows/continuous-integration.yml

```diff
@@ -2,25 +2,14 @@ name: "CI"
 
 on: [push, pull_request]
 
-env:
-  fail-fast: true
```

Comment:
I removed these `fail-fast` parts so tests are not getting canceled and we see if more versions fail.